README.md

```diff
@@ -15,10 +15,10 @@ If wheels aren't getting built, debug over at
 https://github.com/mypyc/mypy_mypyc-wheels/actions
 
 You can use pip to install these wheels like so:
-```
-pip install --upgrade --find-links https://github.com/mypyc/mypy_mypyc-wheels/releases/tag/v0.920+dev.48181d26e7575aece8cab61eb866ac6c573dfd76 mypy
-# or possibly
-pip install --upgrade --find-links https://github.com/mypyc/mypy_mypyc-wheels/releases/latest mypy
+```bash
+pip install --upgrade --find-links https://github.com/mypyc/mypy_mypyc-wheels/releases mypy
```

Comment:
```
👉 pip install --upgrade --find-links https://github.com/mypyc/mypy_mypyc-wheels/releases mypy --dry-run --no-cache --no-index
Defaulting to user installation because normal site-packages is not writeable
Looking in links: https://github.com/mypyc/mypy_mypyc-wheels/releases
Collecting mypy
  Downloading https://github.com/mypyc/mypy_mypyc-wheels/releases/download/v0.990%2Bdev.4ccfca162184ddbc9139f7a3abd72ce7139a2ec3/mypy-0.990%2Bdev.4ccfca162184ddbc9139f7a3abd72ce7139a2ec3-cp310-cp310-manylinux_2_17_x86_64.manyli
nux2014_x86_64.whl (18.2 MB)
```
This is a compiled wheel right? What do you get when you run this?

Reply:
i have the exact same output as @KotlinIsland